docs/Remote-Lakes.md

```diff
@@ -43,8 +41,9 @@ Before diving into the specifics of what's possible, here's an overview of
 some rough edges you may encounter as you work through the configurations
 described in this article.
 
-1. While **logs** can be imported from your local Brim app directly to a remote
-   Lake, **packet captures** currently cannot ([brim/1730](https://github.com/brimdata/brim/issues/1730)).
+1. If a user imports a packet capture while connected to a remote lake, the
+   generated logs are stored remotely but the pcap file and corresponding
+   index for flow extraction remain local to the user's workstation.
```

Comment:
I was actually quite shocked to find this has been already working for so long! See [https://github.com/brimdata/brim/issues/1730#issuecomment-1106699704](https://github.com/brimdata/brim/issues/1730#issuecomment-1106699704) for details.